pipelines/batch/src/main/java/org/openmrs/analytics/FhirEtlOptions.java

```diff
@@ -15,13 +15,14 @@
  */
 package org.openmrs.analytics;
 
+import org.apache.beam.sdk.extensions.gcp.options.GcsOptions;
 import org.apache.beam.sdk.options.Default;
 import org.apache.beam.sdk.options.Description;
 import org.apache.beam.sdk.options.PipelineOptions;
 import org.apache.beam.sdk.options.Validation.Required;
 
 /** Options supported by {@link FhirEtl}. */
-public interface FhirEtlOptions extends PipelineOptions {
+public interface FhirEtlOptions extends PipelineOptions, GcsOptions {
```

Comment:
Question: Why is `GcsOptions` needed here? Is it only because of mocking requirements in tests?

Reply:
GcsOptions is not required. I have removed it in the new commit. Thanks for pointing it out.